util/helpers.go

```diff
@@ -67,3 +68,8 @@ func BuildURL(parts ...string) (string, error) {
 	base.Path = fullPath
 	return base.String(), nil
 }
+
+// TimestampToTime converts a uint32 timestamp to a time.Time object.
+func TimestampToTime(timestamp uint32) time.Time {
```

Comment:
Since `timestamp` will be passed at execution time, we need to check  for a possible overflow here. If a big enough value is passed it will be truncated and will produce an unexpected time.

Reply:
time.Unix() is int64 on Golang, what do you mean by check possible overflow?